spec/spec_helper.rb

```diff
@@ -60,4 +60,7 @@
   end
   config.filter_run focus: true
   config.run_all_when_everything_filtered = true
+  config.before do
+    SidekiqUniqueJobs.config.enabled = false
+  end
```

Comment:
This is recommended with sidekiq-unique-jobs > 6.0.  https://github.com/mhenrixon/sidekiq-unique-jobs/issues/256#issuecomment-407149261